app/models/level.rb

```diff
@@ -8,4 +8,12 @@ class Level < ApplicationRecord
 
   validates_uniqueness_of :mark, scope: :rubric_criterion_id
   validates_numericality_of :mark, greater_than_or_equal_to: 0
+
+  validate :only_update_if_results_unreleased
+
+  def only_update_if_results_unreleased
+    return if self.rubric_criterion.marks.empty?
```

Comment:
Rather than duplicate the logic, reuse the validation method in `RubricCriterion`.